machine-learning/Dockerfile

```
@@ -15,6 +15,34 @@ RUN mkdir /opt/armnn && \
     cd /opt/ann && \
     sh build.sh
 
+# Warning: 25GiB+ disk space required to pull this image
+# TODO: find a way to reduce the image size
+FROM rocm/dev-ubuntu-22.04:6.3.4-complete AS builder-rocm
```

```suggestion
FROM rocm/dev-ubuntu-22.04:6.3.4-complete AS builder-rocm

ARG AMDGPU
ARG HSA_OVERRIDE
ARG HSA_OVERRIDE_GFX_VERSION
```

This might allow us to pass in build args to build for specific architectures and split the image into smaller images. Still testing (waiting for build).

---

Nope. Not it.